avm/res/api-management/service/identity-provider/main.bicep

```diff
@@ -0,0 +1,84 @@
+metadata name = 'API Management Service Identity Providers'
+metadata description = 'This module deploys an API Management Service Identity Provider.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent API Management service. Required if the template is used in a standalone deployment.')
+param apiManagementServiceName string
+
+@description('Optional. Used to enable the deployment of the identityProviders child resource.')
+param enableIdentityProviders bool = false
```

Comment:
The heck? :D Shouldn't that condition be in its parent, if at all?

Reply:
In fact, an interesting implementation. I removed this param and created a condition in the parent.

Reply:
For some odd reason that didn't work which lets me to believe that the tast case is incorrect / some properties that should be mandatory are not. The only case that I could image where this is not the case is if the `identityProviders` parameter is used also for another module deployment - which again would be very odd.

Reply:
This parameter with the default value `false` actually resulted in all IdentityProviders tests being deactivated (which was perhaps even the intention: implement IdentityProviders, but left them out of the test due to the strong integration with AAD). 
But I found out that deploying Identity providers also works with fake values, so I've re-enabled them in the test cases (see [02bfcec](https://github.com/Azure/bicep-registry-modules/pull/713/commits/02bfcecdceadfd3c914f724beb041dba65972cca))